### DISCUSSION TITLE
flashfloppy 3.34 problem

### DISCUSSION THREAD

**Discussion by indiosse** [keirf/flashfloppy#667] [General]
hi with 3.34 updated over 3.33 , my 2 digit lcd stops working with a rotary encoder installed. it works with a 0.91 lcd and rotary but not a 2 digit lcd. any thoughts, its definitely 3.34 causing it though, so i will have to stick with 3.33 until its solved.

**Comment by keirf** [comment]
So it works with the rotary encoder removed?

**Comment by keirf** [comment]
I tested my two 2-digit Goteks with 3.34 just now and they are fine. However they are both esoteric STM32 boards and they do not have rotary encoders. You will need to tell me your board type (name written on PCB), your MCU type (written on the MCU chip), and exactly how/where the rotary encoder is connected.

**Comment by indiosse** [comment]
Ill check and come back to you

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

**Reply by keirf** [reply]
Thanks. If it still fails without rotary encoder that removes one variable, and we can concentrate on board type and chip type. You could also erase cached config by pressing both buttons while no USB stick is inserted for approx 5 seconds.

**Reply by indiosse** [reply]
i did the cache erase and it did work to re-light the screen, i will try with a USB back in there when i get more time. thanks

**Comment by keirf** [comment]
That's good news. If you have a FF.CFG file on your USB stick, here are two recommended setting for a 2-digit display:
```
display-probe-ms = 0
display-type = auto
```

**Reply by indiosse** [reply]
yeah i had already found it wasnt that. ill check the usb loading, and come back if still a problem, ill also take apart and come back with chipset
